tests/test_utils.py

```diff
@@ -75,22 +75,14 @@ def __str__(self):
 
         if self.lineno and hasattr(self, 'source_code'):
 
-            output = 'line %d: %s\n%s' % (
```

Comment:
Can I ask why this was changed?  That string of python source code was just a testing fixture and never gets executed or used for anything.  It's strictly to give the line marker formatter something to test against.

Reply:
I figured that I might as well try to unify as much of the repo as possible on string formatting. Is there a particular issue with updating the test to f-strings?

Reply:
It's just not actually code that's part of the vyper package.  It's just a value that's being used in a test.  It seems like changing it just creates extra work for reviewers and doesn't add any benefit.